### PR TITLE
Fix compiler warnings.

### PR DESCRIPTION
**Pull request by aniemetz**

### CODE REVIEW

**Review by ajreynol — APPROVED** [review]

File: src/theory/arith/theory_arith_private.cpp [hunk 1]
```diff
@@ -3478,7 +3478,6 @@ bool TheoryArithPrivate::postCheck(Theory::Effort effortLevel)
   }
   d_statistics.d_avgUnknownsInARow.addEntry(d_unknownsInARow);
 
-  size_t nPivots = options::useFC() ? d_fcSimplex.getPivots() : d_dualSimplex.getPivots();
```
Comment (mpreiner) [inline src/theory/arith/theory_arith_private.cpp:3481]: Shouldn't `nPivots` be actually the upper bound for the loop below, i.e, `i < nPivots`?

Comment (aniemetz) [reply src/theory/arith/theory_arith_private.cpp:3481]: @nafur

Comment (mpreiner) [reply src/theory/arith/theory_arith_private.cpp:3481]: The code was added here: 
https://github.com/CVC4/CVC4/pull/4763/files#diff-26108011619ab0aae57344114f75c1ad1e596272b61950ab6167ab37c5b1b6a4R3482

Comment (mpreiner) [reply src/theory/arith/theory_arith_private.cpp:3481]: Looks like @nafur forgot to change it to `i < nPivots`

Comment (aniemetz) [reply src/theory/arith/theory_arith_private.cpp:3481]: Addressed in #5495.

**Pull request merged**